lib/github_changelog_generator/generator/generator_fetcher.rb

```diff
@@ -15,6 +15,22 @@ def fetch_events_for_issues_and_pr
       @fetcher.fetch_events_async(@issues + @pull_requests)
     end
 
+    # Determine if a sha has occured as part of a newer tag since an older tag was made.
+    #
+    # @param sha SHA of commit to check.
+    # @param newer_tag The tag to determine if sha is a member.
+    # @param older_tag The tag prior to the newer tag.
+    # @return [Boolean] Whether the sha exists in the newer/older tag comparison.
+    def sha_between_tags?(sha, newer_tag, older_tag)
```

Comment:
This is no longer needed; info is available via `add_first_occurring_tag_to_prs()`